docs/docs/index.md

```diff
@@ -3,12 +3,17 @@ layout: doc-page
 title: "Dotty Documentation"
 ---
 
-The Dotty compiler is currently somewhat lacking in documentation - PRs
-welcome! But, we've attempted to gather the most essential knowledge in these
-pages.
+Dotty is a platform to try out new language concepts and compiler technologies for Scala. 
+The focus is mainly on simplification. We remove extraneous syntax (e.g. no XML literals), 
+and try to boil down Scala’s types into a smaller set of more fundamental constructors. 
+The theory behind these constructors is researched in DOT, a calculus for dependent object types.
+
+In this documentation you will find information on how to use the Dotty compiler on your machine, navigate through
+the code, setup Dotty with your favorite IDE and more!
+
+Contents
+-------
 
-Index
------
 * Usage
```

Comment:
While you're at it, could you add a link to https://github.com/lampepfl/dotty/blob/master/docs/docs/contributing/backend.md in the Contributing section of the table of content? I forgot to add it before.